README.md

```diff
@@ -57,114 +59,97 @@ jobs:
-If dottestcli.exe is not on PATH, you can configure the path to the installation directory of Parasoft dotTEST, which contains dottestcli.exe, by adding the `installDir` parameter do the pre-defined action.
+If `dottestcli` executable is not on `PATH`, you can configure the path to the installation directory of Parasoft dotTEST by configuring the `installDir` parameter:
 
 ```yaml
-- name: Run dotTEST analyzer
-  uses: parasoft/run-dottest-analyzer@1.0.0
+- name: Run dotTEST
+  uses: parasoft/run-dottest-action@1.0.0
   with:
     installDir: 'c:\Program Files\Parasoft\dotTEST\2021.1'
 ```
 
 #### Configuring a dotTEST Test Configuration
-Code analysis with dotTEST is performed by using a test configuration - a set of static analysis rules that enforce best coding practices. Parasoft dotTEST ships with a wide range of [build-in test configurations](https://docs.parasoft.com/display/DOTTEST20202/Built-in+Test+Configurations), as well as allows you to [create your own!](https://docs.parasoft.com/display/DOTTEST20202/Creating+Custom+Test+Configurations)
-To configure a test configuration directly in your workflow, add the `config` parameter to your dotTEST action and specify the URL of the test configuration you want to use:
-
+Code analysis with dotTEST is performed by using a test configuration - a set of static analysis rules that enforce best coding practices. Parasoft dotTEST ships with a wide range of [build-in test configurations](https://docs.parasoft.com/display/DOTTEST20202/Built-in+Test+Configurations).
+To specify a test configuration directly in your workflow, add the `testConfig` parameter to the `Run dotTEST` action and specify the URL of the test configuration you want to use:
 
 ```yaml
 - name: Run dotTEST analyzer
   uses: parasoft/run-dottest-analyzer@1.0.0
   with:
-    config: 'builtin://OWASP Top 10-2017'
+    testConfig: 'builtin://OWASP Top 10-2017'
 ```
 
-Alternatively, you can provide workspace-relative path to the .properties file where your test configuration is defined:
+Alternatively, you can provide the workspace-relative path to the .properties file where your test configuration is defined:
 
 ```yaml
 - name: Run dotTEST analyzer
   uses: parasoft/run-dottest-analyzer@1.0.0
   with:
-    config: '.\.dottest\MyTestConfig.properties'
+    testConfig: '.\.dottest\MyTestConfig.properties'
 ```
 
 #### Defining the Scope for Analysis
-You configure the `solution` parameter to provide the path to the solution you want to analyze. This parameter is particularly useful when your workflow has a complex structure and includes several solution files.
+By default, the `Run dotTEST`action analyzes all solutions in your repository. To modify the default scope for analysis, you can configure the `solution` parameter to provide the path to the solution you want to analyze. This parameter is particularly useful when your workflow has a complex structure and includes several solution files.
```

Comment:
Not precisely... There are many scope switches which are useful in case of complex structure. Also, we do not know if -solution is the right switch for particular user. We need to say something more general here. E.g.: To modify the default scope for analysis, you can configure dotTEST with one of available scope parameters to analyze specific solutions or projects or even selected code files.

Reply:
I've rewritten that section.